### ISSUE TITLE
Cannot read properties of undefined (reading 'transformFile')

### ISSUE THREAD

**Issue by Profsoul** [expo/expo-cli#3985]
### Summary

when i try to execute"npm start" i getting this error often how to overcome this is issue 

### Environment

Node - v17.0.1
Npm - 8.1.0
Os - Ubuntu 21.10

### Please specify your device/emulator/simulator platform, model and version

scaning by expo scanner

### Error output

```
Logs for your project will appear below. Press Ctrl+C to exit.
Failed to construct transformer:  Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:67:19)
    at Object.createHash (node:crypto:130:10)
    at stableHash (/home/soul/Desktop/todo/app1/todoap/node_modules/metro-cache/src/stableHash.js:19:8)
    at Object.getCacheKey (/home/soul/Desktop/todo/app1/todoap/node_modules/metro-transform-worker/src/index.js:647:7)
    at getTransformCacheKey (/home/soul/Desktop/todo/app1/todoap/node_modules/metro/src/DeltaBundler/getTransformCacheKey.js:32:19)
    at new Transformer (/home/soul/Desktop/todo/app1/todoap/node_modules/metro/src/DeltaBundler/Transformer.js:180:28)
    at /home/soul/Desktop/todo/app1/todoap/node_modules/metro/src/Bundler.js:58:29 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
Android Bundling failed 91ms
Cannot read properties of undefined (reading 'transformFile')
```
- [ ] 
![Screenshot from 2021-11-09 20-38-12](https://user-images.githubusercontent.com/61400509/140949894-463a9d4a-a56c-4592-b37c-6384e29c3ac4.png)

### Reproducible demo or steps to reproduce from a blank project

None

**Comment by EvanBacon** [comment]
You must provide "Reproducible demo or steps to reproduce from a blank project", _none_ is not a valid answer.

**Comment by JKojakali** [comment]
You need to change your node version to a more stable version. Anything LTS should work. You can do that by changing the version in your nvmrc file.

**Comment by kbrandwijk** [comment]
See the discussion in https://github.com/expo/expo-cli/issues/3988.

TL;DR: the Metro Bundler does not support Node 17 yet.
